pkg/apis/serving/v1beta1/inference_service_validation_test.go

```diff
@@ -116,78 +116,6 @@ func TestInvalidAutoscalerHPAMetrics(t *testing.T) {
 	g.Expect(isvc.ValidateCreate()).ShouldNot(gomega.Succeed())
 }
 
-func TestValidStorageURIPrefixOK(t *testing.T) {
-	g := gomega.NewGomegaWithT(t)
-	for _, prefix := range SupportedStorageURIPrefixList {
-		isvc := makeTestInferenceService()
-		isvc.Spec.Predictor.Tensorflow.StorageURI = proto.String(prefix + "foo/bar")
-		g.Expect(isvc.ValidateCreate()).Should(gomega.Succeed())
-	}
-}
-
-func TestEmptyStorageURIPrefixOK(t *testing.T) {
-	g := gomega.NewGomegaWithT(t)
-	isvc := makeTestInferenceService()
-	isvc.Spec.Predictor.Tensorflow.StorageURI = proto.String("")
-	g.Expect(isvc.ValidateCreate()).Should(gomega.Succeed())
-}
-
-func TestLocalPathStorageURIPrefixOK(t *testing.T) {
-	g := gomega.NewGomegaWithT(t)
-	isvc := makeTestInferenceService()
-	isvc.Spec.Predictor.Tensorflow.StorageURI = proto.String("some/relative/path")
-	g.Expect(isvc.ValidateCreate()).Should(gomega.Succeed())
-	isvc.Spec.Predictor.Tensorflow.StorageURI = proto.String("/some/absolute/path")
-	g.Expect(isvc.ValidateCreate()).Should(gomega.Succeed())
-	isvc.Spec.Predictor.Tensorflow.StorageURI = proto.String("/")
-	g.Expect(isvc.ValidateCreate()).Should(gomega.Succeed())
-	isvc.Spec.Predictor.Tensorflow.StorageURI = proto.String("foo")
-	g.Expect(isvc.ValidateCreate()).Should(gomega.Succeed())
-}
-
-func TestAzureBlobOK(t *testing.T) {
-	g := gomega.NewGomegaWithT(t)
-	isvc := makeTestInferenceService()
-	isvc.Spec.Predictor.Tensorflow.StorageURI = proto.String("https://kfserving.blob.core.windows.net/triton/simple_string/")
-	g.Expect(isvc.ValidateCreate()).Should(gomega.Succeed())
-	isvc.Spec.Predictor.Tensorflow.StorageURI = proto.String("https://kfserving.blob.core.windows.net/triton/simple_string")
-	g.Expect(isvc.ValidateCreate()).Should(gomega.Succeed())
-	isvc.Spec.Predictor.Tensorflow.StorageURI = proto.String("https://kfserving.blob.core.windows.net/triton/")
-	g.Expect(isvc.ValidateCreate()).Should(gomega.Succeed())
-	isvc.Spec.Predictor.Tensorflow.StorageURI = proto.String("https://kfserving.blob.core.windows.net/triton")
-	g.Expect(isvc.ValidateCreate()).Should(gomega.Succeed())
-}
-
-func TestAzureBlobNoAccountFails(t *testing.T) {
-	g := gomega.NewGomegaWithT(t)
-	isvc := makeTestInferenceService()
-	isvc.Spec.Predictor.Tensorflow.StorageURI = proto.String("https://blob.core.windows.net/triton/simple_string/")
```

Comment:
This uri is actually supported in the new CR because "https" is a valid prefix. Just want to note that this is a behavior change.